<launch>
  <!-- Subscribe to the muxed target topic (real target if available, otherwise a fake target). -->
  <arg name="target_topic" default="/dynamic_map/target_pos" />
  <arg name="map_frame" default="map" />
  <arg name="base_frame" default="base_link" />
  <arg name="follow_distance" default="1.5" />
  <arg name="goal_rate_hz" default="2.0" />

  <node pkg="target_follower" type="follow_target.py" name="target_follower" output="screen">
    <param name="target_topic" value="$(arg target_topic)" />
    <param name="map_frame" value="$(arg map_frame)" />
    <param name="base_frame" value="$(arg base_frame)" />
    <param name="follow_distance" value="$(arg follow_distance)" />
    <param name="goal_rate_hz" value="$(arg goal_rate_hz)" />
  </node>
</launch>
